akka-javasdk/src/main/scala/akka/javasdk/impl/timedaction/TimedActionImpl.scala

```diff
@@ -114,7 +110,7 @@ private[impl] final class TimedActionImpl[TA <: TimedAction](
       case _ =>
         ErrorHandling.withCorrelationId { correlationId =>
           log.error(
-            s"Failure during handling command [${command.name}] from TimedAction component [${command.componentId}].",
+            s"Failure during handling command [${command.name}] from TimedAction component [${timedActionClass.getSimpleName}].",
```

Comment:
in my PR https://github.com/akka/akka-sdk/pull/47 I added the componentId to the TimedActionImpl